test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/CompleteCaseControllerSpec.scala

```diff
@@ -47,9 +48,10 @@ class CompleteCaseControllerSpec extends WordSpec with Matchers with UnitSpec
   private val appConfig = new AppConfig(configuration, env)
   private val casesService = mock[CasesService]
   private val operator = mock[Operator]
+  private val decisionForm = new DecisionForm(new CommodityCodeConstraints(new CommodityCodeService))
```

Comment:
Would it not be more normal to mock the CommodityCodeService?

Reply:
Done